deployment/ccip/changeset/deploy.go

```diff
@@ -422,6 +380,113 @@ func DeployCCIPContracts(
 	return nil
 }
 
+type USDCConfig struct {
+	EnabledChains []uint64
+	USDCAttestationConfig
+	CCTPTokenConfig map[cciptypes.ChainSelector]pluginconfig.USDCCCTPTokenConfig
+}
+
+func (cfg USDCConfig) EnabledChainMap() map[uint64]bool {
+	m := make(map[uint64]bool)
+	for _, chain := range cfg.EnabledChains {
+		m[chain] = true
+	}
+	return m
+}
+
+type USDCAttestationConfig struct {
+	API         string
+	APITimeout  *commonconfig.Duration
+	APIInterval *commonconfig.Duration
+}
+
+type DeployCCIPContractConfig struct {
+	HomeChainSel   uint64
+	FeedChainSel   uint64
+	ChainsToDeploy []uint64
+	TokenConfig    TokenConfig
+	USDCConfig     USDCConfig
+	// For setting OCR configuration
+	OCRSecrets deployment.OCRSecrets
+}
```

Comment:
we need to parameterize the chain configuration values as well those are just hard coded right now https://github.com/smartcontractkit/chainlink/blob/b375b3ca23dd4ff5c4c4524e9773e72ea397b58b/deployment/ccip/changeset/deploy_home_chain.go#L294

Reply:
also I think the OCR timing parameters need to be parameterized and could vary by chain

Reply:
added ticket ccip-4320